README.md

```diff
@@ -65,23 +62,50 @@ fn sum_as_string(a: usize, b: usize) -> PyResult<String> {
     Ok((a + b).to_string())
 }
 
-/// A Python module implemented in Rust.
+/// A Python module implemented in Rust. The name of this function should match
```

Comment:
```suggestion
/// A Python module implemented in Rust. The name of this function must match
```

?

Reply:
Yeah, technically they can write a custom import hook (or use `#[pyo3(name = "foobar")]` to workaround name mismatch, but for a README "must" is probably simpler.